### ISSUE TITLE
NTR: MRI: punctate perivascular enhancement (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9004]
New term request MRI: punctate perivascular enhancement (segal_160811193506): Curvilinear perivascular gadolinium peppering; pons, brainstem, cerebellum  UMLS: 3854437

**Comment by pnrobinson** [comment]
@MickeySegal
this seems to bundle several things. What is the disease context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3927899/
"CLIPPERS: chronic lymphocytic inflammation with pontine perivascular enhancement responsive to steroids. Review of an increasingly recognized entity within the spectrum of inflammatory central nervous system disorders... Characteristic imaging patterns of punctate enhancement involving white and grey matter and neuropathology with prevailing perivascular inflammatory infiltrates favour an inflammatory disorder with a vascular or perivascular tropism"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001037
- label: Punctate pontine perivascular enhancement
- parents: [Abnormal pons morphology (HP:0007361)](https://hpo.jax.org/app/browse/term/HP:0007361)
- pmid: PMID:24028073
- def: Multiple punctate (speckled, patchy spot-like) and/or curvilinear areas of gadolinium enhancement distributed around blood vessels in the pons.
- comment: The lesions may extend into adjacent CNS structures, caudally to the medulla oblongata or the cervicothoracal spinal cord, rostrally to the midbrain, and even involve supratentorial regions such as the thalami, capsula interna, basal ganglia, corpus callosum and cerebral white matter. See Figure 1 of PMID:24028073.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9004
- synonyms:
